src/react-components/src/components/01-atoms/Links/Link.tsx

```diff
@@ -41,8 +42,12 @@ export default class Link extends React.Component<LinkProps, {}> {
       }
     });
 
+    elementChildren.map(child => {
+      return React.cloneElement(child, {key: uid(child)});
```

Comment:
@EdwinGuzman I'd love an extra pair of eyes on this - it seems overkill to use a generator but I'm also not sure what other good candidates there are for keys

Reply:
Hmm I'm not too sure what this does or how it's working. If you're using `.map` then it's returning an array which you're not assigning to anything. So maybe a `forEach` is better. Are you attempting to add a key to each element in `this.props.children`?

Reply:
I'm attempting to add a key to each element in `this.props.children` - creating an element by passing in `children` as array means that React thinks a list is passed, so everything needs a key.  It throws warnings, but I'm not actually sure that the thing it's warning me about is the thing it thinks it's warning me about ...

Reply:
(Or is it that using `React.createElement` with an array of children is an ... anti-pattern?)

Reply:
Short recap of 1-1 talk: I'm not sure if it's an anti-pattern. It may be better to not require the user to pass in an anchor element and instead render it for the user.

Reply:
I made a PR for this issue separately, because it's affecting other work it's better to merge it first -- https://github.com/NYPL/nypl-design-system/pull/73